client/state/data-layer/wpcom/legal/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { registerHandlers } from 'state/data-layer/handler-registry';
+import { LEGAL_REQUEST } from 'state/action-types';
+import { setLegalData } from 'state/legal/actions';
+
+const requestLegalData = action => {
+	return http(
+		{
+			method: 'GET',
+			path: `/legal`,
+			apiNamespace: 'wpcom/v2',
+		},
+		action
+	);
+};
+
+const storeLegalData = ( action, legalData ) => setLegalData( legalData );
+
+const formatLegalData = ( { tos: { accepted, effective_date } } ) => {
+	return {
+		tos: {
+			accepted,
+			effectiveDate: effective_date,
```

Comment:
Just noticed that these have been updated in the API response. It should be `activeDate: active_date`, and the new boolean key `display_prompt`, which is filterable server-side. This `display_prompt` value is what clients should use to determine if a banner should be displayed or not; being filterable, we're using it as a feature flag during testing.

<img width="637" alt="Screen Shot 2020-01-30 at 1 17 43 PM" src="https://user-images.githubusercontent.com/349751/73692898-1893ec00-468a-11ea-801d-78eb5c46ef37.png">

Reply:
Good catch! Updated in last commit